phpunit.xml

```diff
@@ -12,7 +12,7 @@
         <testsuite name="Core">
             <directory>./Modules/Core/Tests/</directory>
         </testsuite>
-        <testsuite name="Dashboard">
+        <!-- <testsuite name="Dashboard">
```

Comment:
Why are these commented?

Reply:
I am refactoring each of them and uncommenting manually to make it compatible for 3.8 testbench. Mostly setUp and tearDown are protected and return void strictly.

Reply:
All are done and file is reverted to its original form